### PR TITLE
feat: added slider

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Added Slider component

### CODE REVIEW

File: components/slider.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  overflow: hidden;
+  -webkit-appearance: none;
+}
+.slider-container input[type="range"]::-webkit-slider-thumb {
+  width: 1rem;
+  -webkit-appearance: none;
+  height: 1rem;
+  background: var(--primary-color);
+  cursor: pointer;
+  box-shadow: -1000px 0 0 1000px var(--slider-track);
```
Comment (Kothalkarpriya) [inline components/slider.css]: you should use rem for more that 10px

**Review by Kothalkarpriya — APPROVED** [review]
After making suggested change you can merge

File: docs/slider.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            <li class="list-inline-item m-4"><a class="link" href="https://twitter.com/bhargavi_s23"><i
+                        class="fab fa-twitter"></i></a></li>
+            <li class="list-inline-item m-4"><a class="link"
+                    href="https://www.linkedin.com/in/bhargavi-sunkireddy-a60b041a1/"><i
+                        class="fab fa-linkedin-in"></i></a></li>
+        </ul>
+        <div class="text-primary pt-3">© 2022 Bhargavi Sunkireddy</div>
+    </footer>
+    <script src="sidebar.js"></script>
+    <script>
```
Comment (Manju-sree) [inline docs/slider.html:87]: keep js scripts in separate folder

**Review by Manju-sree — COMMENTED** [review]
code looks fine to merge

**Pull request merged**